### PR TITLE
Configured requirments.txt

### PR DESCRIPTION
**Pull request by DanBrown47**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,5 @@
-requests
+requests==2.24.0
+argparse==1.4.0
```
Comment (WebBreacher) [inline requirements.txt:2]: Argparse is part of the Python Standard Library and we don't need a specific version.

Running `pipreqs` on my dev system yielded the following requirements file:
```
urllib3==1.25.8
requests==2.22.0
```

File: web_accounts_list_checker_threading.py [hunk 2]
```diff
@@ -81,7 +81,7 @@
 
 def check_os():
     """
-    # check operating system ot adjust output color formatting
+    # check operating system or adjust output color formatting
```
Comment (WebBreacher) [inline web_accounts_list_checker_threading.py:84]: Good catch on this spelling error.

**Review by WebBreacher — CHANGES_REQUESTED** [review]
Can you adjust the requirements.txt file to remove argparse and alter the versions required for the other modules to reflect:

```
urllib3==1.25.8
requests==2.22.0
```

File: requirements.txt (hunk 1 above)
Comment (DanBrown47) [reply requirements.txt]: I have removed the argparse :+1:

**Pull request merged**